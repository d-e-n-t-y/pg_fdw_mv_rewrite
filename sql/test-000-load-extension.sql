create extension pg_fdw_mv_rewrite;
