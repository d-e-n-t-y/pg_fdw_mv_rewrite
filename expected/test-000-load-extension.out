create extension mv_rewrite;
